demo/tekton-demo-setup/03-build-and-deploy.yaml

```diff
@@ -94,7 +94,8 @@ objects:
             set -ex
             find /workspace
             cd /workspace/debug 
-            git log -1 --format=%cd > "$(results.commit-time.path)"
+            epoch_time=`git log -1 --format=%ct`
+            TZ=UTC date -d @`echo $epoch_time` | tee "$(results.commit-time.path)"
```

Comment:
I don't understand this part, I thought the whole point was using epochs?

Also, this line can be simplified into `TZ=UTC date -d @"$epoch_time" | tee "$(results.commit-time.path)"`

Reply:
@KevinMGranger you are not wrong about epochs.  Perhaps after discussing today we'll go for it.  We didn't want to change too much at this time.  I am concerned a little about changing the convert timestamp across all the exporters at this time.   Changing so it just works w/ the committime image exporter is a more smaller change IMHO.